aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -167,56 +167,68 @@ Confirm that the published app for deployment isn't running. Files in the *publi
+![The Publish tab of the app capacities page](visual-studio-publish-profiles/_static/az.png)
 
-Once a publish profile is created, the **Publish** tab changes, and select **Create new profile** to create a new profile.
+When **Folder** is selected, specify a folder path to store the published assets. The default folder is *bin\Release\PublishOutput*. Click the **Create Profile** button to finish.
 
-![The **Publish** tab of the application capacities page showing FolderProfile -created in the last step and Create new profile link](visual-studio-publish-profiles/_static/create_new.png)
+Once a publish profile is created, the **Publish** tab changes. The newly created profile appears in a drop-down list. Click **Create new profile** to create another new profile.
+
+![The Publish tab of the app capacities page showing FolderProfile](visual-studio-publish-profiles/_static/create_new.png)
 
 The Publish wizard supports the following publish targets:
 
-* Microsoft Azure App Service
-* IIS, FTP, etc (for any web server)
+* Azure App Service
+* Azure Virtual Machines
+* IIS, FTP, etc. (for any web server)
 * Folder
-* Import profile (allows profile import).
-* Microsoft Azure Virtual Machines
+* Import Profile
+
+For more information, see [What publishing options are right for me](/visualstudio/ide/not-in-toc/web-publish-options).
 
-See [What publishing options are right for me?](https://docs.microsoft.com/visualstudio/ide/not-in-toc/web-publish-options) for more information.
+When creating a publish profile with Visual Studio, a *Properties/PublishProfiles/&lt;profile_name&gt;.pubxml* MSBuild file is created. The *.pubxml* file is a MSBuild file and contains publish configuration settings. This file can be changed to customize the build and publish process. This file is read by the publishing process. `<LastUsedBuildConfiguration>` is special because it's a global property and shouldn't be in any file that's imported in the build. See [MSBuild: how to set the configuration property](http://sedodream.com/2012/10/27/MSBuildHowToSetTheConfigurationProperty.aspx) for more information.
 
-When creating a publish profile with Visual Studio, a *Properties/PublishProfiles/\<publish name>.pubxml* MSBuild file is created. This *.pubxml* file is a MSBuild file and contains publish configuration settings. This file can be changed to customize the build and publish process. This file is read by the publishing process. `<LastUsedBuildConfiguration>` is special because it's a global property and shouldn't be in any file that's imported in the build. See [MSBuild: how to set the configuration property](http://sedodream.com/2012/10/27/MSBuildHowToSetTheConfigurationProperty.aspx) for more info.
+When publishing to an Azure target, the *.pubxml* file shouldn't be checked into source control. It stores sensitive information, such as the Azure subscription identifier. When publishing to a non-Azure target, it's safer to check in the *.pubxml* file.
```

Comment:
Tthe Azure subscription ID, is the only sensitive thing it stores.  So I would suggest rephrasing it along the lines of "When publishing to an Azure target, the *.pubxml file contains your Azure subscription identifier, so it is not recommended to check it into source control.  When publishing to a non-Azure..."